### PR TITLE
BACK-123 - Init RPC Server and run orchestrator stream

### PR DESCRIPTION
**Pull request by nukky94**
## What is this about?
At the start of the app:
- init and run a RPC server
- init and run the orchestrator stream

## Cute picture of lama
courrier-international-clonage-alpagas

### CODE REVIEW

File: account-manager/src/main/scala/co/ledger/lama/App.scala [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import doobie.util.ExecutionContexts
+import pureconfig.ConfigSource
+
+object App extends IOApp {
+
+  def run(args: List[String]): IO[ExitCode] = {
+    val conf = ConfigSource.default.loadOrThrow[Config]
+
+    val orchestratorResource = for {
+      ce <- ExecutionContexts.fixedThreadPool[IO](32) // our connect EC
```
Comment (RomanWlm) [inline account-manager/src/main/scala/co/ledger/lama/App.scala:15]: Make it configurable

**Comment by vbergeron-ledger** [comment]
Very nice lama !

File: account-manager/src/main/scala/co/ledger/lama/RabbitUtils.scala [hunk 2]
```diff
@@ -15,6 +17,14 @@ import io.circe.{Decoder, Encoder}
 
 object RabbitUtils {
 
+  def createClient(
+      conf: Fs2RabbitConfig
+  )(implicit ce: ConcurrentEffect[IO], cs: ContextShift[IO]): Resource[IO, RabbitClient[IO]] =
+    Resource
+      .make(IO(Executors.newCachedThreadPool()))(es => IO(es.shutdown()))
```
Comment (RomanWlm) [inline account-manager/src/main/scala/co/ledger/lama/RabbitUtils.scala:24]: Asking myself if unbounded pool is really appropriate here. 
Not clear to me

File: account-manager/src/main/scala/co/ledger/lama/App.scala (hunk 1 above)
Comment (vbergeron-ledger) [reply account-manager/src/main/scala/co/ledger/lama/App.scala:15]: A good value here is `Runtime.getRuntime.getAvailableProcessors * 2` in the configuration.

File: account-manager/src/main/scala/co/ledger/lama/RabbitUtils.scala (hunk 2 above)
Comment (nukky94) [reply account-manager/src/main/scala/co/ledger/lama/RabbitUtils.scala:24]: From the scaladoc:
```
     * Creates a thread pool that creates new threads as needed, but
     * will reuse previously constructed threads when they are
     * available.  These pools will typically improve the performance
     * of programs that execute many short-lived asynchronous tasks.
```

On the fs2 rabbit documentation, it also recommends to use this kind of ec

**Pull request merged**